chandra_aca/plot.py

```diff
@@ -119,9 +126,13 @@ def _plot_field_stars(ax, stars, attitude, red_mag_lim=None, bad_stars=None):
         yags, zags = radec2yagzag(stars['RA_PMCORR'], stars['DEC_PMCORR'], quat)
         stars['yang'] = yags * 3600
         stars['zang'] = zags * 3600
+        rows, cols = yagzag_to_pixels(stars['yang'], stars['zang'], allow_bad=True)
+        stars['row'] = rows
+        stars['col'] = cols
```

Comment:
For safety, I think this if block should now also run if 'row' or 'col' is not in colnames.